app/scripts/modules/core/pipeline/config/stages/bake/aws/bakeStage.html

```diff
@@ -71,6 +71,41 @@ <h4 ng-if="viewState.loading" class="text-center">
         <input type="text" class="form-control input-sm"
                ng-model="stage.templateFileName"/>
       </stage-config-field>
+      <stage-config-field label="Extended Attributes" help-key="pipeline.config.bake.extendedAttributes"
+                          ng-if="viewState.roscoMode || (stage.extendedAttributes && _.size(stage.extendedAttributes) > 0)">
```

Comment:
Dang, @duftler merged a moment after I added my comment - I can make the change I suggested in my comment but it would be great if you could do it!